### DISCUSSION TITLE
Playwright template

### DISCUSSION THREAD

**Discussion by svitlanaduman** [Tallyb/cucumber-playwright#459] [General]
Hi Tally, first and foremost thank you for the template! I've also watched a video where you are explaining the details.

I've downloaded an example of a test project directly from playwright, they are providing a package config and examples how to use it with npx and set the running browsers. Have you got a chance to see it? My question is, what the implementation would look like if you created your template based on playwright test project. 
Like first to generate cucumber tests based on feature files and then to run the tests with playwright

I'm trying to wrap my head around it, I thought probably you have this experience.

Best regards
Svetlana

**Comment by Tallyb** [comment]
Hi 
Playwright test and cucumber are 2 different test runners which fulfill the same purpose. Therefore you should use one or the other. 
If you want to use BDD syntax (Gherkin), you should use cucumbers (using this template or creating your own template. This repo is no more than an "idea" of how to do it). 
If you want to use "normal" syntax (describe, test, etc.) for your test - definitely use Playwright test. It is pretty amazing. 
I know there is an open feature request on Playwright repository to support BDD syntax. If this will mature I will definitely move there, but I am not sure IF and WHEN. 

As for using an intermediate generator step - I am not sure this is a good idea from a DX perspective. But I might be wrong. 
I hope this helps.
